product/reports/310_Migration Readiness - Providers/010_VMware Environment Summary.yaml

```diff
@@ -0,0 +1,84 @@
+---
+title: VMware Environment Summary
+rpt_type: Custom
+priority: 10
+db: Host
+cols:
+- name
+- vmm_product
+- vmm_version
+- num_cpu
+- v_total_vms
+include:
+  ext_management_system:
+    columns:
+    - name
+    - api_version
+col_order:
+- ext_management_system.name
+- ext_management_system.api_version
+- name
+- vmm_product
+- vmm_version
+- num_cpu
+- v_total_vms
+headers:
+- Provider
+- vSphere Version
+- Hypervisor
+- Product
+- Version
+- CPU Sockets
+- VMs
+conditions: !ruby/object:MiqExpression
+  exp:
+    "=":
+      field: Host.ext_management_system-type
+      value: ManageIQ::Providers::Vmware::InfraManager
+  context_type:
+order: Ascending
+sortby:
+- ext_management_system.name
+- name
+group: c
+graph:
+dims:
+filename:
+file_mtime:
+categories: []
+timeline:
+template_type: report
+where_clause:
+db_options: {}
+generate_cols:
+generate_rows:
+col_formats:
```

Comment:
also, do we like specifying all these columns to make it easier to copy/paste to a new report, or do we remove fields that are no longer used?

Reply:
I guess we should have modified the report editor after you did the changes that made these fields redundant.